lib/human.py

```diff
@@ -0,0 +1,146 @@
+import io
+import typing
+
+from . import logic, manual_testing
+
+
+# TODO: how should a transpiler handle unexpected values? logging.warning? Throw an exception? Leave a TODO comment in the output?
+
+def main():
```

Comment:
This is just so it's easier to me to play with the library without having to mess with parser.py to try new things. `./dev/main lib/human.py`